py-polars/polars/convert.py

```diff
@@ -50,14 +55,17 @@ def from_dict(
     └─────┴─────┘
 
     """
-    return DataFrame._from_dict(data=data, columns=columns)
+    return DataFrame._from_dict(
+        data=data, columns=columns, schema_overrides=schema_overrides
+    )
 
 
+@deprecated_alias(schema="dtypes")
 def from_dicts(
     dicts: Sequence[dict[str, Any]],
     infer_schema_length: int | None = N_INFER_DEFAULT,
     *,
-    schema: Schema | None = None,
+    schema_overrides: Schema | None = None,
```

Comment:
Functionally, this parameter seems to be equivalent to `schema_overrides`. But we probably will want to add a `schema` parameter too, right?

We may want to do that in a separate PR, but I see that elsewhere you have added both.

Reply:
Yep; I'll add `schema` in the next round (changing `columns`) to bring it up to equivalence.